src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -809,6 +810,12 @@ JitConstants FullyConnected_bf_tiled::GetJitConstants(const fully_connected_para
         jit.AddConstant(MakeJitConstant("TILE_OUT_B_PITCH", params.outputs[0].Batch().pitch));
         jit.AddConstant(MakeJitConstant("BATCH_SIZE", "(OUTPUT_BATCH_NUM)"));
     }
+    auto batch_size = get_input_bf_size(params).first;
+    if (batch_size % (dispatchData.tile_m * dispatchData.tile_ms) != 0) {
```

Comment:
Maybe add stricter condition to avoid performance degradation?
 (tparams.dispatch_bsv == 1) && (batch_size > 1)